wca-guidelines.md

```diff
@@ -151,6 +151,8 @@ To be more informative, each Guideline is classified using one of the following
 - A1a2+) [ADDITION] In case of a cumulative time limit, the judge records the original recorded time for a DNF on the score sheet in parentheses, e.g. "DNF (1:02.27)".
 - A1a2++) [CLARIFICATION] Cumulative time limits may be enforced for a single round or a combination of rounds of different events.
 - A1a2+++) [CLARIFICATION] In case of a cumulative time limit, the result of each remaining attempt in the round is recorded as DNS (Did Not Start) after a competitor reaches the time limit.
+- A1a2++++) [ADDITION] If cumulative time limits for a single event or a combination of events are enforced, such events don't have regular time limits per solve.
+- A1a2+++++) [CLARIFICATION] Events where time limit per solve is defined by Regulations (Fewest Moves Solving and Multiple Blindfolded Solving) can't have cumulative time limits.
```

Comment:
I think this should be either "a time limit" or "time limits".

Reply:
Also, we should take into account #405 here and name the events correctly.

Reply:
Nit: "cannot" instead of "can't".